### DISCUSSION TITLE
Modify the snap camera app to never disable the filter

### DISCUSSION THREAD

**Discussion by Hypnotistchris** [ptrumpis/snap-camera-server#31] [Ideas]
Hi
Is there anyway you could modify the snap camera app to never disable the filter?
Would be happy to pay for some development time 😊
Thanks

**Comment by ptrumpis** [comment]
No, but you can achieve similar behavior with these steps:

- Assign hotkeys to your filters
- Run snap camera on system startup
- Automatically trigger the hotkey on system startup (delayed)

You will probably have to find a third party tool to auto trigger the hotkey or use a scripting language.

**Reply by Hypnotistchris** [reply]
Hi
I already do this. The issue I have is that one mistake of pressing the same one hotkey twice and the filter is removed.
Looking for a way round it. As I said would be happy to pay for the right person to modify and remove the deactive from the exe
Thanks
